testing/features/support/components/header/standard.rb

```diff
@@ -8,10 +8,10 @@ class Standard < ::Base
     include Helpers::Javascript
 
     element :change_language, "a", text: "Cymraeg"
-    element :login, "[data-testid='account-link']"
```

Comment:
We probably don't want to undo data id hooking if possible. Also the data id probably still exists.

Reply:
**Sign in / Sign out element**
```
<li class="cads-header__account-link">
    <form action="/sign-out/out" class="cads-header__account-form" method="POST">
        <button class="cads-linkbutton__regular">Sign out</button>
    </form>
</li>
```

Expecting for login (element :login, "[data-testid='account-link']")
`expect(@component).to have_login`

Generate the following error:
```
RSpec::Expectations::ExpectationNotMetError: expected #<Header::Standard:0x00007fa2161373f0 @loaded=true, @load_error=nil> to have login
...
/Users/marcelonicolosi/Projects/design-system/testing/features/step_definitions/header_steps.rb:12:in `"a login/logout link is present"'
/Users/marcelonicolosi/Projects/design-system/testing/features/header.feature:26:in `a login link is present'
```

Reply:
Looking at the haml it's likely we just need to add the test id into both branches of the if statement.

Reply:
I didn't fully get it! Add the data-testid again on the front-end?

Reply:
Yep, I'll give you a call this morning if you need help